dist/ansible/ovn-uninstall.yaml

```diff
@@ -34,13 +34,23 @@
   # iptables -D INPUT -p tcp -m state --state NEW -m tcp --dport 6641 -j ACCEPT
   - name: iptables allow 6641
     shell: iptables -D INPUT -p tcp -m tcp --dport 6641 -m conntrack --ctstate NEW -j ACCEPT
+    ignore_errors: yes
 
   # iptables -D INPUT -p tcp -m state --state NEW -m tcp --dport 6642 -j ACCEPT
   - name: iptables allow 6642
     shell: iptables -D INPUT -p tcp -m tcp --dport 6642 -m conntrack --ctstate NEW -j ACCEPT
+    ignore_errors: yes
 
 # - name: Add adm policy auyuid
 #   shell: oc adm policy add-scc-to-user anyuid -z ovn
 #   ignore_errors: yes
 
 
+- hosts: nodes
+  become: yes
+  become_user: root
+  tasks:
+  - name: Remove remove the cni plugin config
```

Comment:
We probably just want to remove the config file we drop in there, not the whole directory...

Reply:
@dcbw We explicitly create the directory, and the file. That is why I deleted it. Would you like me to change it?

Reply:
@dcbw I made the change. PTAL